control-plane-operator/controllers/hostedcontrolplane/cloud/azure/providerconfig.go

```diff
@@ -43,8 +43,19 @@ func ReconcileCloudConfigWithCredentials(secret *corev1.Secret, hcp *hyperv1.Hos
 		return err
 	}
 
-	cfg.AADClientID = hcp.Spec.Platform.Azure.ManagedIdentities.ControlPlane.CloudProvider.ClientID
-	cfg.AADClientCertPath = config.ManagedAzureCertificatePath + hcp.Spec.Platform.Azure.ManagedIdentities.ControlPlane.CloudProvider.CertificateName
+	// Get the ClientID and CertificateName config for csi drivers in ARO
+	switch secret.Name {
+	case "azure-disk-csi-config":
```

Comment:
can we please get the name cases from the library funcs, e.g. manifests.AzureFileConfigWithCredentials("")?

Reply:
fixed, thanks!